ui/components/ui/export-text-container/export-text-container.component.js

```diff
@@ -2,60 +2,60 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useI18nContext } from '../../../hooks/useI18nContext';
 import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
-import { exportAsFile } from '../../../helpers/utils/export-utils';
-import { Icon, ICON_NAMES } from '../../component-library';
+import {
+  ALIGN_ITEMS,
+  BORDER_COLORS,
+  BORDER_RADIUS,
+  DISPLAY,
+  FLEX_DIRECTION,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
+import Box from '../box/box';
+import { ButtonSecondary, Text, TEXT_VARIANTS } from '../../component-library';
 
-function ExportTextContainer({
-  text = '',
-  onClickCopy = null,
-  onClickDownload = null,
-}) {
+function ExportTextContainer({ text = '', onClickCopy = null }) {
+  const ONE_MINUTE = 1000 * 60;
   const t = useI18nContext();
-  const [copied, handleCopy] = useCopyToClipboard();
+  const [copied, handleCopy] = useCopyToClipboard(ONE_MINUTE);
 
   return (
-    <div className="export-text-container">
-      <div className="export-text-container__text-container">
-        <div className="export-text-container__text notranslate">{text}</div>
-      </div>
-      <div className="export-text-container__buttons-container">
-        <div
-          className="export-text-container__button export-text-container__button--copy"
-          onClick={() => {
-            if (onClickCopy) {
-              onClickCopy();
-            }
-            handleCopy(text);
-          }}
-        >
-          <Icon name={copied ? ICON_NAMES.COPY_SUCCESS : ICON_NAMES.COPY} />
-          <div className="export-text-container__button-text">
-            {copied ? t('copiedExclamation') : t('copyToClipboard')}
-          </div>
-        </div>
-        <div
-          className="export-text-container__button"
-          onClick={() => {
-            if (onClickDownload) {
-              onClickDownload();
-            }
-            exportAsFile('', text);
-          }}
-        >
-          <img src="images/download.svg" alt="" />
-          <div className="export-text-container__button-text">
-            {t('saveAsCsvFile')}
-          </div>
-        </div>
-      </div>
-    </div>
+    <Box
+      display={DISPLAY.FLEX}
+      justifyContent={JUSTIFY_CONTENT.CENTER}
+      flexDirection={FLEX_DIRECTION.COLUMN}
+      alignItems={ALIGN_ITEMS.CENTER}
+      borderColor={BORDER_COLORS.BORDER_DEFAULT}
+      borderRadius={BORDER_RADIUS.MD}
+      padding={4}
+      gap={4}
+    >
+      <Text
+        display={DISPLAY.FLEX}
+        justifyContent={JUSTIFY_CONTENT.CENTER}
+        className="notranslate"
+        variant={TEXT_VARIANTS.BODY_LG}
+      >
+        {text}
+      </Text>
+      <ButtonSecondary
+        className="export-text-container__button"
+        block={true}
```

Comment:
I think eslint is giving an error and suggesting
```suggestion
        block
```